sdk/src/client/stronghold/mod.rs

```diff
@@ -52,14 +52,14 @@ mod migration;
 mod secret;
 mod storage;
 
+use alloc::sync::Weak;
 use std::{
     path::{Path, PathBuf},
     sync::Arc,
     time::Duration,
 };
 
-use derive_builder::Builder;
```

Comment:
Can the dependency be removed? I think it was the only use

Reply:
I think it would tell me if it was unused.

Reply:
Oh, misread

Reply:
Idk, udeps is notoriously full or misses and false positives. It really looks unused to me